07/hmw-07.py

```diff
@@ -0,0 +1,86 @@
+phone_book = dict()
+
+# create the initial phone book
+while True:
```

Comment:
This part is redundant since you can still work with an empty phonebook and fill it in the next loop. I would suggest removing it.

Reply:
Hi, I have commented the 1st loop which was to create the phone book